run/secrets/auth_cookie_key

```diff
@@ -0,0 +1 @@
+auth-dev-cookie-key
```

Comment:
Are these files artifacts?

Reply:
No, I added these files to start successfully the prod-setup. Without these files, the prod-setup won't start because the `KeyService` cannot find these files.

Reply:
Please move them together with the docker-compose.yml into an own folder.